modules/RTC/ScreenObtainer.js

```diff
@@ -389,6 +392,19 @@ const ScreenObtainer = {
                 /* eslint-enable no-alert */
             }
 
+            // for opera there is no inline install
+            // extension "Download Chrome Extension" allows us to open
+            // the chrome webstore and install from there and then activate our
+            // extension
+            if (RTCBrowserType.isOpera()) {
+                options.listener('waitingForExtension',
+                    getWebStoreInstallUrl(this.options));
+                this.checkForChromeExtensionOnInterval(
+                    options, streamCallback, failCallback);
+
```

Comment:
Maybe it will be nice if we create a function ( handleExternalInstall or something like that) for this 2 lines.